src/detectors/LOWQ2/LOWQ2.cc

```diff
@@ -0,0 +1,66 @@
+// Copyright 2023, Simon Gardner
+// Subject to the terms in the LICENSE file found in the top-level directory.
+//
+//
+
+#include <Acts/Propagator/Navigator.hpp>
+
+#include <JANA/JApplication.h>
+#include <JANA/JEvent.h>
+
+#include "extensions/jana/JChainMultifactoryGeneratorT.h"
+
+
+#include <factories/digi/SiliconTrackerDigi_factoryT.h>
+#include <factories/fardetectors/FarDetectorTrackerCluster_factoryT.h>
+#include <factories/fardetectors/FarDetectorLinearTracking_factoryT.h>
+#include <factories/fardetectors/FarDetectorMLReconstruction_factoryT.h>
+
+
+extern "C" {
+  void InitPlugin(JApplication *app) {
+    InitJANAPlugin(app);
+
+    using namespace eicrecon;
+
+    // -------------------------------
+    // TODO: Separate algorithms from factories and add config parameters
+    // -------------------------------
+
+    // Digitization
+    SiliconTrackerDigiConfig digi_cfg;
+    //digi_cfg.timeResolution = 2.5; // Change timing resolution.
+
+    //Clustering config
+    FarDetectorTrackerClusterConfig cluster_cfg;
+    //Ensure same detector is passed to digi and clustering
+    cluster_cfg.readout = "TaggerTrackerHits";
```

Comment:
Preferred is:
```suggestion
    FarDetectorTrackerClusterConfig cluster_cfg{
      //Ensure same detector is passed to digi and clustering
      .readout = "TaggerTrackerHits",
    };
```
or even directly into the lines below without naming a variable.